README.md

```diff
@@ -1 +1,177 @@
+Inference intelligently utilizes heterogeneous hardware, including GPUs and CPUs, to maximize
+performance and accelerate your model inference tasks.
+
+⚙️ **Flexible API and Interfaces**: Xorbits Inference offers multiple interfaces for interacting
+with your models. You can utilize the RPC and RESTful API(compatible with OpenAI API) to integrate
+your models with existing systems or use the command-line interface (CLI) and the intuitive WebUI
+for seamless management and monitoring.
+
+🌐 **Distributed Deployment**: Xorbits Inference excels in distributed deployment scenarios, 
+allowing the seamless distribution of model inference across multiple devices or machines. It
+leverages distributed computing techniques to parallelize and scale the inference process.
+
+🔌 **Built-in Integration with Third-Party Libraries**: Xorbits Inference provides seamless
+integration with popular third-party libraries like LangChain and LlamaIndex. (Coming soon)
+
+## Getting Started
+Xinference can be installed via pip from PyPI. It is highly recommended to create a new virtual
+environment to avoid conflicts.
+```bash
+$ pip install xinference
+```
+
+### Deployment
+To start a local instance of Xinference, run the following command:
+
+```bash
+$ xinference -H,--host "localhost" \
+             -p,--port 9997 \
+             --log-level INFO
+```
+
+To deploy Xinference in a cluster, you need to start an Xinference supervisor on one server and 
+Xinference workers on the other servers. Follow the steps below:
+
+#### Starting the Supervisor
+On the server where you want to run the Xinference supervisor, run the following command:
+```bash
+$ xinference-supervisor -H,--host "${supervisor_host}" \
+                        -p,--port 9997 \
+                        --log-level INFO
+```
+Replace `${supervisor_host}` with the actual host of your supervisor server.
+
+#### Starting the Workers
+On each of the other servers where you want to run Xinference workers, run the following command:
+```bash
+$ xinference-worker -e, --endpoint "http://${supervisor_host}:9997" \
+                    -H,--host "0.0.0.0" \
+                    --log-level INFO
+```
+
+Once Xinference is running, an endpoint will be accessible for model management via CLI or
+Xinference  client.
+
+- For local deployment, the endpoint will be `http://localhost:9997`.
+- For cluster deployment, the endpoint will be `http://${supervisor_host}:9997`, where
+`${supervisor_host}` is the hostname or IP address of the server where the supervisor is running.
+
+You can also view a web UI using the Xinference endpoint where you can chat with all the 
+builtin models. You can even **chat with two cutting-edge AI models side-by-side to compare
+their performance**!
+
+![web UI](assets/xinference-downloading.png)
+
+### Xinference CLI
+Xinference provides a command line interface (CLI) for model management. Here are some useful 
+commands:
+
+- Launch a model: `xinference launch`
```

Comment:
Shall us tell users that a `model_uid` will be provided?

Reply:
Done.